.github/workflows/publish-storage-browser.yml

```diff
@@ -64,6 +64,6 @@ jobs:
     uses: ./.github/workflows/reusable-tagged-publish.yml
     needs: e2e
     with:
-      dist-tag: storage-browser-alpha
+      dist-tag: storage-browser
```

Comment:
not following this change. Don't we want to publish to alpha on the main branch?